src/ptracer.rs

```diff
@@ -343,6 +489,9 @@ impl Ptracer {
                 self.remove_tracee(pid);
                 return self.wait();
             },
+            WaitStatus::Stopped(pid, SIGINT) => {
```

Comment:
Is this related to the rest of the PR? What's motivating this change?

Reply:
Woops. That slipped in. It's not related to the rest of the PR. I'll excise it. It is important and I'll open a separate PR for this. It makes it possible to catch `SIGINT`s. Without this, they are swallowed up.